.github/workflows/pr_check.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           with:
             python-version: ${{ matrix.python-version }}
         - name: Install dependencies
-          run: python -m pip install --upgrade "tox>=4.0.9,<5" build
+          run: python -m pip install --upgrade "tox>=4.0.9,<5" build lark
```

Comment:
Do we need `lark` package here?
During the tox run, we do `pip install -r requirements.txt` that will handle this install